src/KlaviyoAPI.php

```diff
@@ -179,7 +179,7 @@ private function handleResponse( $response, $statusCode, $isPublic )
         } else if ( $statusCode == 404 ) {
             throw new KlaviyoResourceNotFoundException(self::ERROR_RESOURCE_DOES_NOT_EXIST);
         } else if ( $statusCode == 429 ) {
-            throw new KlaviyoRateLimitException( $this->decodeJsonResponse( $response ) );
+            throw new KlaviyoRateLimitException( $response );
```

Comment:
Only thought here would be to decode the response and specifically pull out the time until retry

Reply:
yea, we could definitely just do:
`$this->decodeJsonResponse( $response )['detail']` instead?

Reply:
Yea, I think that's cleaner but I guess pulling the retry seconds from a json string versus the message string is the same.

Reply:
the new return will look like:
{"detail":"Request was throttled. Expected available in 26.0 seconds.","retryAfter":26}